components/org.wso2.carbon.identity.auth.otp.core/src/main/java/org/wso2/carbon/identity/auth/otp/core/constant/AuthenticatorConstants.java

```diff
@@ -60,6 +60,10 @@ public class AuthenticatorConstants {
     public static final String MULTI_OPTION_URI_PARAM = "&multiOptionURI=";
     public static final String RECAPTCHA_PARAM = "&reCaptcha=";
     public static final String USERNAME_PARAM = "&username=";
+    public static final String PROPERTY_LOGIN_FAIL_TIMEOUT_RATIO = "account.lock.handler.login.fail.timeout.ratio";
```

Comment:
Can you check whether it is possible to use the `org.wso2.carbon.identity.handler.event.account.lock.constants.AccountConstants` instead of redefining these constants here?

Reply:
fixed